xskillscore/core/comparative.py

```diff
@@ -0,0 +1,160 @@
+import warnings
+
+import scipy.stats as st
+import xarray as xr
+
+
+def sign_test(
+    forecast1,
+    forecast2,
+    observation=None,
+    time_dim=None,
```

Comment:
Should this default to `time`? What happens with it as `None`? I see that it's used in `cumsum` and so on.

Reply:
Time would be the most realistic. If none it's applied over all dimensions.